### PR TITLE
Bump to kind v0.11.0

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

File: files/common/scripts/setup_env.sh [hunk 1]
```diff
@@ -63,7 +63,7 @@ fi
 
 # Build image to use
 if [[ "${IMAGE_VERSION:-}" == "" ]]; then
-  export IMAGE_VERSION=master-2021-05-27T17-28-48
+  export IMAGE_VERSION=master-2021-05-27T21-48-48
```
Comment (mandarjog) [inline files/common/scripts/setup_env.sh:66]: Can you point to build tools commit that has version 0.11.0?

Comment (howardjohn) [reply files/common/scripts/setup_env.sh:66]: https://github.com/istio/tools/pull/1553/files

**Review by mandarjog — APPROVED** [review]
Lgtm

**Comment by ericvn** [comment]
Note that we are currently using kind 0.11.1 with the latest image.

**Review by ericvn — CHANGES_REQUESTED** [review]
We have moved to a newer version of kind, so merging this will revert that change.

**Pull request closed without merging**